__tests__/utils.test.js

```diff
@@ -102,3 +104,26 @@ describe("formatComments", () => {
     expect(formattedComments[0].created_at).toEqual(new Date(timestamp));
   });
 });
+
+describe("compareDates - compare callback for the sort array method", () => {
```

Comment:
You shouldn't sort with utils functions. Use SQL query to sort by created_at in descending order.